simulator/risc_v/t/unit_test.cpp

```diff
@@ -285,6 +285,45 @@ TEST_CASE( "RISV RV64 xnor")
     CHECK( instr.get_v_dst( 0) == 0x6fff'ffff'6fff'ffff);
 }
 
+TEST_CASE( "RISV RV32 max")
+{
+    CHECK( RISCVInstr<uint32>(0x0ae7d7b3).get_disasm() == "max $a5, $a5, $a4");
+    RISCVInstr<uint32> instr( "max", 0);
+    instr.set_v_src( 0x1111'1111, 0);
+    instr.set_v_src( 0xf111'111f, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst( 0) == 0x1111'1111);
+}
+
+TEST_CASE( "RISV RV64 max")
+{
+    RISCVInstr<uint64> instr( "max", 0);
+    instr.set_v_src( 0xffff'ffff'ffff'ffff, 0);
+    instr.set_v_src( 0x1111'1111'3333'3333, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst( 0) == 0x1111'1111'3333'3333);
+}
+
+TEST_CASE( "RISV RV32 maxu")
+{
+    CHECK( RISCVInstr<uint32>(0x0ae7f7b3).get_disasm() == "maxu $a5, $a5, $a4");
+    RISCVInstr<uint32> instr( "maxu", 0);
+    instr.set_v_src( 0xffff'ffff, 0);
+    instr.set_v_src( 0x1111'1111, 1);
```

Comment:
Could you please use same values for `max` and `maxu` so difference would be explicitly visible?